fullscreen.bs

```diff
@@ -180,8 +190,8 @@ partial interface DocumentOrShadowRoot {
      interface, which is implemented by Document and HTMLElement, not Element. -->
 
 <dl class=domintro>
- <dt><code><var>promise</var> = <var>element</var> . {{Element/requestFullscreen()}}</code>
- <dd><p>Displays <var>element</var> fullscreen and resolves <var>promise</var> when done.
+ <dt><code><var>promise</var> = <var>element</var> . {{Element/requestFullscreen(<var>options</var>)}}</code>
+ <dd><p>Displays <var>element</var> fullscreen with <var>options</var> and resolves <var>promise</var> when done.
```

Comment:
This should describe the options members in a way that conveys the meaning to web developers.